### PR TITLE
Add sidekiq 6.2 to appraisals.

### PR DESCRIPTION
**Pull request by jmpage**
## What did we change?

Test against sidekiq 6.2 for supported rails versions.

## Why are we doing this?

Addresses #46 

Sidekiq recently released a new minor revision. To protect against compatibility breaks between this gem and the new version of sidekiq, we want to test against that version of the gem.

## How was it tested?
- [x] Specs
- [ ] Locally
- [ ] Staging

### CODE REVIEW

File: gemfiles/rails_6.0_sidekiq_6.2.gemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "activejob", "~> 6.0.0"
+gem "activesupport", "~> 6.0.0"
+gem "sidekiq", "~> 6.1.0"
```
Comment (tjwp) [inline gemfiles/rails_6.0_sidekiq_6.2.gemfile:7]: This one says "6.1.0"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
   be breaking but it does represent a major change in the gem's implementation
   and that is the reason for the major version bump.
 - Require Ruby 2.6 or later.
+- Test against sidekiq 6.2
```
Comment (tjwp) [inline CHANGELOG.md:10]: I don't think this needs a changelog entry. 

And now I realize that I forgot to remove `(unreleased)` from the changelog. This has actually been released. 🤦

**Review by tjwp — CHANGES_REQUESTED** [review]
Thanks for picking this up!

File: CHANGELOG.md (hunk 2 above)
Comment (tjwp) [reply CHANGELOG.md:10]: Pushed a commit to master to fix this.

File: gemfiles/rails_6.0_sidekiq_6.2.gemfile (hunk 1 above)
Comment (jmpage) [reply gemfiles/rails_6.0_sidekiq_6.2.gemfile:7]: Thanks Tim. I noticed the typeo in the Appraisals file but forgot to reinstall them.

Comment (jmpage) [reply gemfiles/rails_6.0_sidekiq_6.2.gemfile]: Just force pushed a fix.

File: CHANGELOG.md (hunk 2 above)
Comment (jmpage) [reply CHANGELOG.md]: Removed

**Review by tjwp — APPROVED** [review]

**Pull request merged**